docker-app/qfieldcloud/core/management/commands/extractstoragemetadata.py

```diff
@@ -0,0 +1,127 @@
+import csv
+import logging
+import sys
+from pathlib import Path
+from typing import Generator, NamedTuple
+
+import boto3
+import mypy_boto3_s3
+import yaml
+from django.core.management.base import BaseCommand, CommandParser
+from qfieldcloud.core import utils
+
+logger = logging.getLogger(__name__)
+
+
+class Command(BaseCommand):
+    """Save metadata from S3 storage project files to disk."""
+
+    def add_arguments(self, parser: CommandParser):
+        parser.add_argument("-o", "--output", type=str, help="Name of output file")
```

Comment:
I think we can print to the `stdout` by default, so the `docker compose exec python manage.py` command can be redirected?